app/src/org/commcare/activities/CommCareWiFiDirectActivity.java

```diff
@@ -822,22 +822,20 @@ private void updateStatusText() {
 
     public static boolean copyFile(InputStream inputStream, OutputStream out) {
         Logger.log(TAG, "File server copying file");
-        Log.d(CommCareWiFiDirectActivity.TAG, "Copying file");
         if (inputStream == null) {
-            Log.d(CommCareWiFiDirectActivity.TAG, "Input Null");
+            Logger.log(TAG, "Input stream null");
+            return false;

```

Comment:
This looks like a functional change, not a logging change?

Reply:
I've updated the behavior to be more correct, nice catch